components/brave_wallet_ui/common/trezor/trezor_bridge_keyring.ts

```diff
@@ -97,6 +120,21 @@ export default class TrezorBridgeKeyring extends EventEmitter {
     return ''
   }
 
+  private prepareTransactionPayload = (path: string, txInfo: TransactionInfo, chainId: string): SignTransactionCommandPayload => {
+    return {
+      path: path,
+      transaction: {
+        to: txInfo.txData.baseData.to,
+        value: txInfo.txData.baseData.value,
+        data: bufferToHex(Buffer.from(txInfo.txData.baseData.data)).toString(),
+        chainId: parseInt(chainId, 16),
+        nonce: txInfo.txData.baseData.nonce,
+        gasLimit: txInfo.txData.baseData.gasLimit,
+        gasPrice: txInfo.txData.baseData.gasPrice
```

Comment:
Are we not supporting EIP-1559 gas pricing fields like `maxFeePerGas` and `maxPriorityFeePerGas`?

Reply:
added